### PR TITLE
OpenVINO at the openSUSE Factory - cabelo@opensuse.org

### PR DESCRIPTION
**Pull request by cabelo**
@tadamczx and @ilya-lavrenov  It took a while but I delivered what was promised, openVINO at the official openSUSE factory. It took more than 6 months of work to get approval from the community...

### CODE REVIEW

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
build_jenkins

**Review by sgolebiewski-intel — APPROVED** [review]

File: docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst [hunk 1]
```diff
@@ -48,23 +48,16 @@ Install OpenVINO™ Runtime on Linux From ZYPPER Repository
...
-      sudo zypper addrepo https://download.opensuse.org/repositories/science/openSUSE_Tumbleweed/science.repo
       sudo zypper refresh
 
-2. Verify that the new repository is set up properly.
-
-   .. code-block:: sh
-
-      zypper lr |grep -i science
-
    You will see the available list of packages.
```
Comment (sgolebiewski-intel) [inline docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst:61]:
```suggestion
   You will see the available list of packages.
```

File: docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst [hunk 2]
```diff
@@ -48,24 +48,16 @@ Install OpenVINO™ Runtime on Linux From ZYPPER Repository
...
 
-2. Verify that the new repository is set up properly.
-
-   .. code-block:: sh
-
-      zypper lr |grep -i science
-
-   You will see the available list of packages.
 
 To list available OpenVINO packages, use the following command:
```
Comment (sgolebiewski-intel) [inline docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst:62]:
```suggestion
2. To list available OpenVINO packages, use the following command:
```

**Review by sgolebiewski-intel — APPROVED** [review]

**Comment by sgolebiewski-intel** [comment]
I have applied a few minor changes. @cabelo Thank you very much for this update.

**Comment by cabelo** [comment]
@sgolebiewski-intel , @ilya-lavrenov , @adk9 Thank you, I'm happy to complete this work.

Do I need to do anything else?

**Comment by ilya-lavrenov** [comment]
build_jenkins

**Pull request merged**